### DISCUSSION TITLE
既然愚人节版本被单独分类了，那么为什么远古版不被单独分类呢

### DISCUSSION THREAD

**Discussion by misividkoukou** [Meloong-Git/PCL#3706] [提问] [answered]
感觉现在启动器的版本分类方式十分繁杂，版本一安装多了就很乱，不知道可不可以有一种方式把版本分类做得更好一点？

**Comment by allMagicNB** [comment]
远古版一般不会作为正常游玩使用（除了考古学家们），但是愚人节版会

**Reply by misividkoukou** [reply]
那么预览版不单独分类如何解释呢？

**Reply by allMagicNB** [reply, marked as answer]
> 那么预览版不单独分类如何解释呢？
> 

快照版一般玩新不玩旧，旧的不常用，所以只筛选一个到常用版本中

**Comment by MoYuan-CN** [comment]
![image](https://github.com/Hex-Dragon/PCL2/assets/109946426/f687a346-33ec-4c10-b2ba-54d2a1d639a2)
……？

**Reply by allMagicNB** [reply]
喂，Author 说的是版本选择页面啊

**Reply by misividkoukou** [reply]
这不是下载页面吗......

**Comment by iPodtouch6557** [comment]
哦对了，搜索版本也在投票了
